Code/GraphMol/ChemTransforms/catch_tests.cpp

```diff
@@ -383,6 +383,15 @@ TEST_CASE("molzip", "[]") {
     auto b = "[*:1]-N=C"_smiles;
     auto mol = molzip(*a, *b, p);
   }
+
+  SECTION("MolZip saves bonddir") {
```

Comment:
This isn't testing what you just changed. Bond 0 isn't the one added by the zipping code.
If I comment out your change in `MolFragmenter.cpp` this test still passes.

Reply:
Good catch on that one.